### PR TITLE
Improve the examples

### PR DESCRIPTION
**Pull request by jkoplo**
* Add more line prints and comments
* Add an example for array read

### CODE REVIEW

File: src/Examples/CSharp DotNetFramework/Program.cs [hunk 1]
```diff
@@ -1,40 +1,18 @@
-﻿using libplctag;
-using System;
-using System.Net;
-using System.Threading;
+﻿using System;
 
-namespace CSharpDotNetFramework
+namespace CSharpDotNetCore
```
Comment (timyhac) [inline src/Examples/CSharp DotNetFramework/Program.cs:3]: @jkoplo this one is in a .NET framework project - was this intentional?

**Comment by timyhac** [comment]
Nice work @jkoplo

File: src/Examples/CSharp DotNetFramework/Program.cs (hunk 1 above)
Comment (jkoplo) [reply src/Examples/CSharp DotNetFramework/Program.cs:3]: Oops. I copied and pasted from the other one. There's probably a better way to just have one example project that switches platform with a compilation target (`AnyCPU-Core`) - I think I've seen that done before.

Comment (timyhac) [reply src/Examples/CSharp DotNetFramework/Program.cs:3]: Oh cool I haven't seen that before.

**Pull request merged**